### PR TITLE
kubectl rabbitmq plugin

### PR DESCRIPTION
**Pull request by mkuratczyk**
* moved from our internal repo
* new commands: install-cluster-operator, secrets, debug, observe
* small behaviour changes

To test:
1. `export PATH=~/workspace/cluster-operator/bin:$PATH`
2. `kubectl rabbitmq` (it will display help)

You can see a demo here: https://asciinema.org/a/354729

### CODE REVIEW

File: bin/kubectl-rabbitmq [hunk 1]
```diff
@@ -0,0 +1,304 @@
...
+        kubectl delete rabbitmqcluster "${cluster}"
+    done
+}
+
+observe() {
+    kubectl exec -it "${1}-rabbitmq-server-${2}" -- rabbitmq-diagnostics observer
+}
+
+get() {
+    kubectl get pods,cm,sts,svc,secrets,rs -l "app.kubernetes.io/name=$1"
```
[the remark this answers is not in the document]
Comment (mkuratczyk) [reply bin/kubectl-rabbitmq:174]: I decided not to. It's a lot of YAML that is unlikely to affect anything (as in, there is no runtime state and it's basically impossible to see there is anything wrong with it). I think there should be another command, something like "collect support data" that should collect everything indeed, including `sa,roles,rolesbindings`

File: bin/kubectl-rabbitmq [hunk 2]
```diff
@@ -0,0 +1,304 @@
...
+    password=$(kubectl get secret "${instance}-rabbitmq-admin" -o jsonpath="{.data.password}" | base64 --decode)
+    service=${instance}-rabbitmq-client
+}
+
+perf_test() {
+    get_instance_details "$@"
+    shift 1
+    perftestopts=$*
+
+    kubectl run perf-test \
```
[the remark this answers is not in the document]
Comment (mkuratczyk) [reply bin/kubectl-rabbitmq:76]: I haven't made any changes. When I add `--generator=run-pod/v1` I get `Flag --generator has been deprecated, has no effect and will be removed in the future.` so I don't know what the correct command is but the current one works...

File: bin/kubectl-rabbitmq (hunk 1 above)
[the remark this answers is not in the document]
Comment (ansd) [reply bin/kubectl-rabbitmq:174]: Okay, sounds good to me. I just wasn't sure whether they were intentionally left out.

File: bin/kubectl-rabbitmq (hunk 2 above)
[the remark this answers is not in the document]
Comment (ansd) [reply bin/kubectl-rabbitmq:76]: https://stackoverflow.com/a/53010591
It depends on the K8s version whether a deprecation warning is printed.
When I tested with 1.18, it only created a pod without deprecation warning.
Right, let's leave this for now as-is and not block the PR on it.

**Pull request merged**